File: sum of squares of first n natural numbers.py
# Python3 Program to 
# find sum of square 
# of first n natural  
# numbers 
  
  
# Return the sum of 
# square of first n 
# natural numbers 
def squaresum(n) : 
   
    sm = 0
    for i in range(1, n+1) : 
        sm = sm + (i * i)  
    return sm 
  
# Driven Program 
n = input("enter the number upto which you want the sum of squares")
print(squaresum(n)) 

File: sum_of_squares_of_first_n_natural_numbers.c
#include<stdio.h>
int sumofsquares(int n)
{
    return (n * (n + 1) *  (2 * n + 1)) / 6;
}
int main()
{
    int n;
    printf("Please enter the number of terms : ");
    scanf("%d",&n);
    printf("The sum of squares of the first n natural numbers is : %d\n",sumofsquares(n));
}